src/test/java/teammates/test/pageobjects/InstructorHomePage.java

```diff
@@ -164,6 +164,30 @@ public InstructorHomePage clickFeedbackSessionPublishLink(String courseId, Strin
         return changePageType(InstructorHomePage.class);
     }
 
+    public void clickResendPublshedLinksLink(String courseId, String evalName) {
+        click(getResendPublishedLinksLink(courseId, evalName));
+        ThreadHelper.waitFor(1000);
```

Comment:
Is there something concrete we can wait for?

Reply:
There are a few methods to wait for modal presence:
https://github.com/TEAMMATES/teammates/blob/497bf6da33843286a45bf99e3ec646c6a9c3ec2c/src/test/java/teammates/test/pageobjects/AppPage.java#L325-L338
None of them work in this case though.

Reply:
Have you tried experimenting with `waitForPageToLoad` (it may wait for the AJAX request to complete though I'm not sure) or `waitForAjaxLoaderGifToDisappear` (less direct but still better than a fixed wait)? Can fix `clickRemindParticularUsersLink` too.

Reply:
Yeah I already tried both of them. What's odd is that I also tried `waitForElementPresence` with the `id` of the modal itself and that didn't work. `waitForElementVisibility` works tho..